test/deploy-prepare-and-test-pipeline.sh

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env bash
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+PRJ_ROOT="$(cd `dirname "${BASH_SOURCE}"`/..; pwd)"
+
+ENV_CODE=${1:-${ENV_CODE}}
+
+BATCH_ACCOUNT_NAME=${2:-${BATCH_ACCOUNT_NAME:-"stellarbatchdev"}}
+PIPELINE_NAME=${3:-${PIPELINE_NAME:-"custom-vision-model-v2"}}
+
+DESTINATION_CONTAINER_NAME=${4:-${DESTINATION_CONTAINER_NAME:-"${PIPELINE_NAME}-${ENV_CODE}"}}
+DESTINATION_STORAGE_ACCOUNT_NAME=${5:-${DESTINATION_STORAGE_ACCOUNT_NAME}}
+DESTINATION_STORAGE_ACCOUNT_RG_NAME=${6:-${DESTINATION_STORAGE_ACCOUNT_RG_NAME:-"${ENV_CODE}-data-rg"}}
+SOURCE_CONTAINER_NAME=${7:-${SOURCE_CONTAINER_NAME:-${PIPELINE_NAME}}}
+SOURCE_STORAGE_ACCOUNT_NAME=${8:-${SOURCE_STORAGE_ACCOUNT_NAME:-"stellarciresources"}}
```

Comment:
Same here. Remove references to our internal batch accounts. Also, we should remove all references to "stellar".